SegmentEditorDrawTube/SegmentEditorDrawTubeLib/SegmentEditorEffect.py

```diff
@@ -280,20 +280,23 @@ def reset(self):
       self.setAndObserveSegmentMarkupNode(None)
 
   def onApply(self):
-    # Allow users revert to this state by clicking Undo
-    self.scriptedEffect.saveStateForUndo()
-
-    # This can be a long operation - indicate it to the user
-    qt.QApplication.setOverrideCursor(qt.Qt.WaitCursor)
-    self.observeSegmentation(False)
-    self.logic.cutSurfaceWithModel(self.segmentMarkupNode, self.segmentModel)
-    self.reset()
-    # Create model node prior to markup node for display order
-    self.createNewModelNode()
-    self.createNewMarkupNode()
-    self.fiducialPlacementToggle.setCurrentNode(self.segmentMarkupNode)
-    self.observeSegmentation(True)
-    qt.QApplication.restoreOverrideCursor()
+    if self.getNumberOfDefinedControlPoints() >= 3:

```

Comment:
no need for this long indentation here, just return immediately if self.getNumberOfDefinedControlPoints() < 3